config/environments/development.rb

```diff
@@ -1,15 +1,6 @@
 Rails.application.configure do
-  # Settings specified here will take precedence over those in config/application.rb.
-
-  # In the development environment your application's code is reloaded on
-  # every request. This slows down response time but is perfect for development
-  # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
-
-  # Do not eager load code on boot.
   config.eager_load = false
-
-  # Show full error reports.
```

Comment:
This was an attempt to bypass a rubocop lint.
Forgot to revert, but ultimately don't think we need these comments, so leaving it out.